### PR TITLE
Add my signature cookies!

### PR DESCRIPTION
**Pull request by tamarbenjoseph**
Love you!

### CODE REVIEW

File: tamars_cookies.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Ingredients
```
Comment (grantseltzer) [inline tamars_cookies.md]: If you look in the `challah.md` file, Ingredients should be lower case, use two `#`'s (meaning a 2nd tier header), and the name of the recipe should be at the top with a single `#`

File: tamars_cookies.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
```
Comment (grantseltzer) [inline tamars_cookies.md]: Can you remove extra empty lines? There should just be a single one in between each sections/titles/etc...

File: tamars_cookies.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+30 grams Dutch-processed cocoa powder
+¾ teaspoon baking powder
+½ teaspoon kosher salt
+225 grams bittersweet chocolate (70 percent), broken or chopped into pieces
+55 grams unsalted butter (1/2 stick)
+2 large eggs, at room temperature
+100 grams granulated sugar
+55 grams light brown sugar
+1 teaspoon vanilla extract
+115 grams bittersweet chocolate chips or chopped chocolate (about 2/3 cup)
```
Comment (grantseltzer) [inline tamars_cookies.md]: Consider adding bakers percentages following the format that we have in the challah recipe.

File: tamars_cookies.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Ingredients
+
+
+
+```
+45 grams all-purpose flour
+30 grams Dutch-processed cocoa powder
```
Comment (grantseltzer) [inline tamars_cookies.md]: Can you also add a new line in between each ingredient?

File: tamars_cookies.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+¾ teaspoon baking powder
+½ teaspoon kosher salt
```
Comment (grantseltzer) [inline tamars_cookies.md]: Nit: Can you replace the fraction symbols with decimals? 

i.e.) ½ -> 0.5

**Review by grantseltzer — CHANGES_REQUESTED** [review]
Looks like a delicious recipe. In addition to the changes I request in review comments, can you also  change the file name to something more descriptive?

Suggestions: `double-chocolate-cookies.md` or `chocolate-crumble-cookies.md`

Also, do you want to keep ginger in this recipe?

File: tamars_cookies.md (hunk 2 above)
Comment (grantseltzer) [inline tamars_cookies.md:4]:
```suggestion
```

Comment (grantseltzer) [inline tamars_cookies.md:4]:
```suggestion
# Ingredients
```

File: tamars_cookies.md (hunk 3 above)
Comment (tamarbenjoseph) [reply tamars_cookies.md]: Nahhhh

File: tamars_cookies.md (hunk 5 above)
Comment (tamarbenjoseph) [reply tamars_cookies.md]: No.

**Comment by grantseltzer** [comment]
LGTM!

**Pull request merged**